Refine documentation (#132)
iter8-tools/docs: docs/getting-started/advantages.md

---

# Advantages

## Use any resources

> Iter8 allows an application to be composed of resources of any types, including custom resources; that is, those defined by custom resource definitions (CRDs). In Iter8, the set of resources that make up an application version is declarative making it easy to extend for new resource types. The same extension mechanism also allows Iter8 to be used with any service mesh or ingress.

Reviewer: ```suggestion
Iter8 allows an application to be composed of resources of any type, including custom resources defined by custom resource definitions (CRDs). In Iter8, the set of resources that make up an application version is declarative making it easy to extend for new resource types. The same extension mechanism also allows Iter8 to be used with any service mesh or ingress.
```

Reviewer: I think the first sentence having a semi colon clause makes it clunky to read, which isn't great for the first sentence of this page.

## A/B/n testing of backend components

> Using the Iter8 SDK to develop frontend application components enables them to reliably associate business metrics with the contributing version of the backend. This addresses a key challenge encountered when doing [A/B/n testing](../user-guide/abn/about.md) of backend application components/ML models.

Reviewer: ```suggestion
Using the Iter8 SDK enables frontend application components to reliably associate business metrics with the contributing versions of the backend. This addresses a key challenge of [A/B/n testing](../user-guide/abn/about.md) of backend application components/ML models.
```

The first sentence is quite long so trying to cut words.

## Simplified performance testing

> Iter8 reduces set up time to begin running performance tests. Tests can be easily specified as a sequence of [easily configured tasks](../user-guide/performance/parameters.md). Further, there is no need to setup and configure an external metrics database -- Iter8 captures the metrics data and provides a REST API to access it, allowing it to be visualized and evaluated in Grafana.

Reviewer: I think the original reads as a little clunky. There are also two instances of the word "easily" in the same sentence. I'm not entirely happy with this version either.

# Comparison to other tools 

> [Flagger](https://flagger.app/) and [Argo Rollouts](https://argo-rollouts.readthedocs.io/en/stable/) share similarities with Iter8. Both provide support advanced application rollout on Kubernetes with blue-green and canary analysis. They work with many service meshes and ingress products to provide this support. Users specify the desired rollout using a Kubernetes custom resource.

Reviewer: ```suggestion
[Flagger](https://flagger.app/) and [Argo Rollouts](https://argo-rollouts.readthedocs.io/en/stable/) share similarities with Iter8. Both provide support for advanced application rollout on Kubernetes with blue-green and canary analysis. They work with many service meshes and ingress products to provide this support. Users specify the desired rollout using a Kubernetes custom resource.
```
